packages/Ludown/lib/ludown-parse-ToLuis.js

```diff
@@ -33,6 +33,13 @@ program
 if (process.argv.length < 4) {
     program.help();
 } else {
+    if (program.luis_culture) {
+        // List of supported LUIS.ai locales
+        const LUISLocales = ['en-us', 'fr-ca', 'zh-cn', 'nl-nl', 'fr-fr', 'de-de', 'it-it', 'ja-jp', 'ko-kr', 'pt-br', 'es-es', 'es-mx'];
```

Comment:
This list is strange, but seems to match the LUIS docs:
https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-language-support

I might add a link to the docs into the code.